swift-mode.el

```diff
@@ -583,11 +583,15 @@
                       (string :tag "Linked Sources"))
        :safe #'stringp)
 
+     (flycheck-def-option-var flycheck-swift-framework-search-paths nil swift
+       "A list of framework search paths")
+
```

Comment:
It may be worth to add a sanity check here with `:safe #'flycheck-string-list-p`, what do you think?